feat: change discord link to feedback form

benfurber opened the pull request:
PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [x] New feature (non-breaking change which adds functionality)


## Description

This changes the current button (that's on every page) that goes to discord to a quick feedback form using retool.

I think it's best to keep it super simple atm, see if we 

## Git Issues

Closes #3716

benfurber commented:
@davehakkens The form can be editted here: https://onearmy.retool.com/forms/editor/Reporting/CP%20-%20Feedback%20Form

benfurber commented:
Also. Un-hide the button on mobile/tablet - worth checking it looks ok so have added a preview to review.

davehakkens commented:
Nice to use retool for this @benfurber to keep it simple
- Who do you imagine screening/sorting the feedback? dev team/moderators?
- Would it be possible to add a label per instance? So we know from which instance the feedback comes in
- Agree with @pizzaisdavid in the issue that 'Report a problem" would be better wording
- Not sure sure about mobile. The button is quite prominent there, but not a strong opinion on it. Also up for trying.

benfurber commented:
@davehakkens 

* Screening: Dev team
* Label per instance: I've changed url to auto-populate, do you think that does the trick?
* Updated wording
* Mobile: Trying a smaller versioin, what do you think?

davehakkens commented:
Nice. Yes all good to me, the auto url approuch is even better!
But doesnt work here `unknown`, but perhaps due to test environment.
<img width="313" alt="afbeelding" src="https://github.com/ONEARMY/community-platform/assets/13672737/31704a24-0313-4cfc-9937-c1f77af7d2d3">

benfurber commented:
> Nice. Yes all good to me, the auto url approuch is even better! But doesnt work here `unknown`, but perhaps due to test environment. <img alt="afbeelding" width="313" src="https://private-user-images.githubusercontent.com/13672737/345056154-31704a24-0313-4cfc-9937-c1f77af7d2d3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk5Mjg2NzAsIm5iZiI6MTcxOTkyODM3MCwicGF0aCI6Ii8xMzY3MjczNy8zNDUwNTYxNTQtMzE3MDRhMjQtMDMxMy00Y2ZjLTk5MzctYzFmNzdhZjdkMmQzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzAyVDEzNTI1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVlMzU3MzFjZGE0Y2E0YmE1OTNiMDU4YmU3ZWU5MGI1NjhiYjRiNTg1YjViNDQ5NTM0ZTQ4ZTg1NmRkNjVhZjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.J8OxyMOVukUYCq_C9erEoIz9FyMq8HkTgfjgrmMlEIY">

Hhmmm, I was hoping that the preview branch would have that set the config field correctly... I'll have another go.

benfurber commented:
@davehakkens In fact I'm confident enough I've got the full url that I've locked the field (if the page loads with a url provided as a param).
